recipes/cobra/meta.yaml

```diff
@@ -0,0 +1,53 @@
+package:
+  name: cobra
+  version: "0.4.1"
+
+source:
+  fn: cobra-0.4.1.tar.gz
+  url: https://pypi.python.org/packages/source/c/cobra/cobra-0.4.1.tar.gz
+  md5: a96799117abed427333bb57e007a8b82
+
+build:
```

Comment:
Can we add the following after `build:`?

    skip: true  # [win and py3k]

It seems that `glpk` is not getting built for the newer versions of `vc*`, which is required to get it to work with newer version of Python (they must use the versions of the MSVC library). That is why the Python 3.x builds are failing on Windows. We can go take a look `glpk` and see why this is.